xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -20,6 +20,7 @@
 #include "pvr/channels/PVRChannel.h"
 #include "pvr/epg/EpgInfoTag.h"
 #include "pvr/recordings/PVRRecording.h"
+#include "pvr/recordings/PVRRecordings.h"
```

Comment:
Not needed, slipped in by accident most likely

Reply:
It's needed. => https://github.com/xbmc/xbmc/pull/15707/files#diff-66efc2be2d45b88d3b10d92bb3e36139R120

Reply:
But now you have it twice? Does the diff play tricks on me?

Reply:
???

```
#include "pvr/recordings/PVRRecording.h"
#include "pvr/recordings/PVRRecordings.h"
#include "pvr/recordings/PVRRecordingsPath.h"
```

no dups here.

Reply:
the "**s**" at the end :-) was it, sorry for the noise

Reply:
np, thanks for taking a look